assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -64,9 +59,14 @@ export const TakeActionBoxoutEditor = ({
 
     // eslint-disable-next-line no-shadow
     const actPageList = [].concat(
-      select('core').getEntityRecords('postType', 'page', args) || [],
-      select('core').getEntityRecords('postType', 'p4_action', actionsArgs) || []
-    ).sort((a, b) => {
+      select('core').getEntityRecords('postType', 'page', {
+        ...args, parent: window.p4_vars.options.take_action_page,
```

Comment:
This still won't work with the old IA, because then we need to use `act_page` instead

Reply:
I've pushed a new change!